### ISSUE TITLE
Publish native modules with name that follows Netty's naming scheme

### ISSUE THREAD

**Issue by slandelle** [hyperxpro/Brotli4j#6]
Hi @hyperxpro !

I'd like to work on PR for Netty to bring Brotli decoder support.

However, the current naming scheme makes it very cumbersome (I would have to play with profiles to map the netty scheme to brotly4j's current one.

Would it be possible to change it?

* `native-macos_x86-64` => `native-osx-x86_64`
* `native-linux_x86-64` => `native-linux-x86_64`
* `native-windows_x86-64` => `native-windows-x86_64`

Basically:
* `macos` => `osx`
* `x86-64` => `x86_64` (underscore)

**Comment by hyperxpro** [comment]
Looks good. :D 

Can you do PR? I am stuck with something these days.

**Comment by hyperxpro** [comment]
Also, should we move to 2.X instead of changing this in 1.X?

**Comment by slandelle** [comment]
PTAL https://github.com/hyperxpro/Brotli4j/pull/7
